src/CAREUI/display/RelativeTime.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react";
+import moment from "moment";
+import CareIcon from "../icons/CareIcon";
+
+interface Props {
+  time?: string;
+  prefix?: React.ReactNode;
+  className?: string;
+  user?: {
+    first_name: string;
+    last_name: string;
+    last_login?: string;
+  };
+}
+
+/**
+ * A generic component to display relative time along with a tooltip and a user
+ * if provided.
+ */
+const RelativeTime = ({ time, user, prefix, className }: Props) => {
+  const [relativeTime, setRelativeTime] = useState(moment(time).fromNow());
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setRelativeTime(moment(time).fromNow());
+    }, 1000);
```

Comment:
just a thought, instead of having this 1000 (1s) why not have it dynamic meaning,
if it is 5 minutes ago, we can have 60 * 1000
if it is hours, it could be 60 * 60 * 1000